tests/test_manifest.py

```diff
@@ -48,8 +47,7 @@ def setUp(self):
 		super(TestManifestInclusion, self).setUp()
 		# Arguments need to build
 		clean_arguments = [
-			str("{} -m coverage run").format(sys.executable),
-			'setup.py', 'clean', '--all'
+			str("{} -m coverage run").format(sys.executable), 'setup.py', 'clean', '--all'
```

Comment:
This could also be improved as per #134 with F-strings.

```suggestion
			f"{str(sys.executable)} -m coverage run", "setup.py", "clean", "--all"
```